File: frontend/src/app/models/DTO/Common/buildDTO.ts
/* Auto Generated */

import { BuildStatus } from "./../../Enums/buildStatus"
import { ProjectDTO } from "./projectDTO"
import { UserDTO } from "./../User/userDTO"

export interface BuildDTO {
    buildMessage: string;
    buildStarted: Date;
    buildFinished?: Date;
    buildStatus: BuildStatus;
    uriForArtifactsDownload: string;
    projectId: number;
    project: ProjectDTO;
    userId: number;
    user: UserDTO;
}

File: backend/IDE.Common/ModelsDTO/DTO/Common/BuildDTO.cs
using IDE.Common.DTO.User;
using IDE.Common.Enums;
using System;

namespace IDE.Common.DTO.Common
{
    public class BuildDTO
    {
        public string BuildMessage { get; set; }
        public DateTime BuildStarted { get; set; }
        public DateTime? BuildFinished { get; set; }
        public BuildStatus BuildStatus { get; set; }
        public int ProjectId { get; set; }
        public ProjectDTO Project { get; set; }
        public int UserId { get; set; }
        public UserDTO User { get; set; }
    }
}
